build_all.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# this script replaces the all.yml playbook due to a variable scoping issue
+# that appeared with Ansible 2.0.0.2
+SCRIPT_DIR=$(cd $(dirname $0) && pwd)
```

Comment:
What is this for? Also, I don't think we really need this comment. The commit message/PR explains it for anyone who is really curious.